### ISSUE TITLE
Implementation in pytorch

### ISSUE THREAD

**Issue by AshishRMenon** [pathak22/context-encoder#50]
Any idea on how to implement this paper in pytorch would be of a great help.!
And what do these line of the code do?
How do I implement these lines in pytorch

real_ctx[{{},{1},{},{}}][mask_global] = 2*117.0/255.0 - 1.0
real_ctx[{{},{2},{},{}}][mask_global] = 2*104.0/255.0 - 1.0
real_ctx[{{},{3},{},{}}][mask_global] = 2*123.0/255.0 - 1.0
input_ctx:copy(real_ctx)

**Comment by russab0** [comment]
There is a [GitHub repository](https://github.com/eriklindernoren/PyTorch-GAN), you may get some insights from that. But there are some differences with the original code (tt least, num_epochs and reconstruction loss differ)
